File: back/FilmesApi/FilmesApi/Models/Participante.cs
using System;
using System.Collections.Generic;

namespace FilmesApi.Models;

public partial class Participante
{
    public int Id { get; set; }

    public string Nome { get; set; } = null!;

    public string CNPJ { get; set; }

    public string RazaoSocial { get; set; }

    public string Email { get; set; } = null!;

    public string Senha { get; set; } = null!;

    public string Telefone{ get; set; }

    public string Tipo { get; set; } = null!;

    public DateTime DataRegistro { get; set; }

    public virtual ICollection<Empresa> Empresas { get; set; } = new List<Empresa>();
}

File: front/src/app/models/participante.ts
import { Empresa } from './empresa'; // Importe o tipo Empresa

export class Participante {
    id?: number;
    nome?: string;
    CNPJ?: string;
    razaoSocial?: string;
    email?: string;
    senha?: string;
    telefone?: string;
    tipo?: string;
    dataRegistro?: Date;
    empresas?: Empresa[];
}

